src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -186,7 +188,7 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
     });
     this.onContactTypeValueChanges();
     this.enableSignUpSubmitButton();
-    this.onPhoneChange();
+    // this.onPhoneChange();
```

Comment:
can you please remove this line as this is no longer needed

Reply:
👍